incubator/kafka/values.yaml

```diff
@@ -7,9 +7,9 @@ Image: "solsson/kafka"
 ImageTag: "0.11.0.0"
 ImagePullPolicy: "IfNotPresent"
 Cpu: "100m"
-Memory: "512Mi"
+Memory: "1024Mi"
```

Comment:
Raising default limit were needed because Kafka will require 1GB of RAM for JVM Heap only, and some more memory for other JVM areas, probes, etc. I had OOM kills for Kafka pods while testing our software.

Nevertheless, I reworked it a little to use resource: {}, but haven't tested yet. Is this a right way?